### ISSUE TITLE
Grails 3 support

### ISSUE THREAD

**Issue by imikh** [mathifonseca/grails-version-update#5]

**Comment by antony** [comment]
```
ant@vore:~/Projects/grails-sendgrid$ grails version-update M
| Error Command not found version-update
Did you mean: create-integration-test or create-controller or create-unit-test?
```

**Comment by mathifonseca** [comment]
Hi @antony, did you run a `grails compile --refresh-dependencies` before that?

**Comment by antony** [comment]
Hi

I did, yeah. I ran them separately, though.

**Comment by mathifonseca** [comment]
How strange then. Which version of Grails are you using?

**Comment by antony** [comment]
I'm using the very latest one.

You can check out my repo desirable-objects/grails-sendgrid if you want to
try to reproduce - that's the project in trying to use it on. I've kept it
installed there.

**Comment by mathifonseca** [comment]
Ok, this plugin doesn't support Grails 3 yet. I'll work on that today and let you know when it's ready :)

**Comment by antony** [comment]
Superb, thanks!

**Comment by mathifonseca** [comment]
@antony I've just released version `2.0.0` for Grails 3.x. I've submitted the request to add it to the Grails official repository, but if you want to give it a try right now, you can add my Bintray to your dependencies: https://bintray.com/mathifonseca/grails-plugins. Please read the new instructions in the repo home about the configuration for Grails 3 apps.

**Comment by antony** [comment]
Great, thanks!
